services/web/server/src/simcore_service_webserver/studies_dispatcher/_constants.py

```diff
@@ -1,13 +1,16 @@
 from typing import Final
 
-# NOTE: MSG_* strings MUST be human readable messages
+# NOTE: MSG_$(ERROR_CODE_NAME) strings MUST be human readable messages
 
-MSG_PROJECT_NOT_FOUND: Final[str] = "Cannot find any study with ID '{project_id}'."
 
+MSG_PROJECT_NOT_FOUND: Final[str] = "Cannot find any study with ID '{project_id}'"
 
-MSG_PROJECT_NOT_PUBLISHED: Final[
+
+MSG_PROJECT_NOT_PUBLISHED: Final[str] = "Cannot find any study with ID '{project_id}'"
```

Comment:
I'd rephrase to: "You need to be logged in to access study with ID '{project_id}'. Please login and try again. \n If you don't have an account, write to the Support email to request one".

Reply:
This error happens when the linked study doe not exists or is not shared with everyone